packages/pyright-internal/src/tests/hoverProvider.test.ts

```diff
@@ -438,3 +438,133 @@ test('hover on __call__ method', async () => {
+        marker3: [
+            '```python\n(method) def __add__(self: Self@int, value: int, /) -> int\n```',
+            { start: { line: 7, character: 0 }, end: { line: 7, character: 6 } },
+        ],
+        marker4: [
+            '```python\n(method) def __invert__(self: Self@int) -> int\n```',
+            { start: { line: 8, character: 5 }, end: { line: 8, character: 7 } },
+        ],
+        marker5: [
+            '```python\n(method) def __and__(self: Self@int, value: int, /) -> int\n```',
+            { start: { line: 8, character: 5 }, end: { line: 8, character: 13 } },
+        ],
+        marker6: [
+            '```python\n(method) def __rmul__(self: Self@tuple[_T_co@tuple], value: SupportsIndex, /) -> tuple[_T_co@tuple, ...]\n```',
+            { start: { line: 9, character: 4 }, end: { line: 9, character: 9 } },
+        ],
+        marker7: [
+            '```python\n(method) def __contains__(self: Self@tuple[_T_co@tuple], key: object, /) -> bool\n```',
+            { start: { line: 10, character: 4 }, end: { line: 10, character: 14 } },
+        ],
+        marker8: [
+            '```python\n(method) def __lt__(self: Self@int, value: int, /) -> bool\n```',
+            { start: { line: 11, character: 4 }, end: { line: 11, character: 12 } },
+        ],
+        marker9: [
+            '```python\n(method) def __getitem__(self: Self@tuple[_T_co@tuple], key: SupportsIndex, /) -> _T_co@tuple\n```',
+            { start: { line: 11, character: 8 }, end: { line: 11, character: 12 } },
+        ],
+        marker10: null,
+        marker11: [
+            '```python\n(method) def __bool__(self: Self@int) -> bool\n```',
+            { start: { line: 12, character: 9 }, end: { line: 12, character: 14 } },
+        ],
+        marker12: [
+            '```python\n(method) def __setitem__(self: Self@list[_T@list], key: SupportsIndex, value: _T@list, /) -> None\n```',
+            { start: { line: 14, character: 0 }, end: { line: 14, character: 26 } },
+        ],
+        marker13: [
+            '```python\n(method) def __setitem__(self: Self@list[_T@list], key: SupportsIndex, value: _T@list, /) -> None\n```',
+            { start: { line: 14, character: 11 }, end: { line: 14, character: 26 } },
+        ],
+        marker14: [
+            '```python\n(method) def __getitem__(self: Self@list[_T@list], i: SupportsIndex, /) -> _T@list\n```',
+            { start: { line: 14, character: 22 }, end: { line: 14, character: 26 } },
+        ],
+        marker15: [
+            '```python\n(variable) b: float\n```\n\n\n---\n```python\n(method) def __setitem__(self: Self@dict[_KT@dict, _VT@dict], key: _KT@dict, value: _VT@dict, /) -> None\n```',
+            { start: { line: 18, character: 0 }, end: { line: 18, character: 15 } },
+        ],
+        marker16: [
+            '```python\n(variable) a: int\n```\n\n\n---\n```python\n(method) def __getitem__(self: Self@dict[_KT@dict, _VT@dict], key: _KT@dict, /) -> _VT@dict\n```',
```

Comment:
i think the hover text for these `TypedDict` keys should say "(key)" instead of "(variable)". that will make it consistent with the hover text on the definition:

<img width="240" height="65" alt="Image" src="https://github.com/user-attachments/assets/0073f3bc-a132-4d4e-a7c7-fb3f953498c8" />

```suggestion
        marker15: [
            '```python\n(key) b: float\n```\n\n\n---\n```python\n(method) def __setitem__(self: Self@dict[_KT@dict, _VT@dict], key: _KT@dict, value: _VT@dict, /) -> None\n```',
            { start: { line: 18, character: 0 }, end: { line: 18, character: 15 } },
        ],
        marker16: [
            '```python\n(key) a: int\n```\n\n\n---\n```python\n(method) def __getitem__(self: Self@dict[_KT@dict, _VT@dict], key: _KT@dict, /) -> _VT@dict\n```',
```

Reply:
Done.